.github/workflows/tests.yml

```diff
@@ -20,5 +20,5 @@ jobs:
           group: 'unit tests'
         env:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CYPRESS_myToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
myToken seems vague...can we rename it back to something that tells us it's for GH?

Reply:
We can. For context, I chose this name from reviewing the example code from [@actions/core usage](https://github.com/actions/toolkit/tree/main/packages/github#usage)